src/app/components/pool/action/add/pool-create-basic-dialog.html

```diff
@@ -21,7 +21,8 @@
             </div>
         </bl-form-section>
 
-        <bl-form-section title="Scale" subtitle="Number of nodes in your pool using a fixed number or an auto scaling forumla">
+        <bl-form-section title="Scale" subtitle="Number of nodes in your pool using a fixed number or an auto scaling formula">
+            <a (click)="openLink('https://docs.microsoft.com/en-us/azure/batch/batch-low-pri-vms')" href="javascript:void(0)">Learn more about Spot/Low Priority Nodes</a>
```

Comment:
Remove "/en-us" so it's language neutral.

Reply:
Updated